packages/rum-core/src/domain/plugins.ts

```diff
@@ -0,0 +1,27 @@
+import type { RumPublicApi } from '../boot/rumPublicApi'
+import type { RumInitConfiguration } from './configuration'
+
+export interface RumPlugin {
+  name: string
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Maybe adding the plugin version, for telemetry and maybe checking compatibility with the SDK.

Reply:
For now installing a plugin with a different version than the SDK won't be supported. Plugins and SDK need to be updated at the same time.